AnonymousAuth/Podfile

```diff
@@ -1,4 +1,4 @@
-platform :ios, '8.0'
+platform :ios, '9.0'
```

Comment:
the core sdk currently support ios 8. If it's going to be bumped to 9.0(most likely), this is fine. Otherwise, we may want to keep FirebaseUI consistent with the core sdk.

Reply:
Yeah, I mentioned this to Paul and he said bumping to 9.0 makes sense.

Reply:
Sounds good.

Reply:
we ended up bumping to 10 for Firebase 7. Probably should do that here too.

Reply:
Done.